controlpanel.user.js

```diff
@@ -24,7 +24,19 @@
 
 (function() {
     var menuTableTools = document.getElementById('menuTableTools');
-    var menuGTNG = document.createElement("li");
-    menuGTNG.innerHTML = '<a class="menubutton" href="#"><span class="textlabel">GalaxytoolNG</span></a>'
+    var menuGTNG = document.createElement('li');
+    function toogleMenu() {
+        var bottonz = document.getElementById('buttonz');
+        bottonz.style.display = 'none';
+        var settingsGTNG = document.createElement('div');
+        settingsGTNG.id = 'settingsGTNG';
+        settingsGTNG.innerHTML = '<div class="header c-left"><h2>GalaxytoolNG settings:</h2></div><div class="content">HELLO</div>';
+        document.getElementById("contentWrapper").appendChild(settingsGTNG);
+    }
+    menuGTNG.innerHTML = '<a class="menubutton" href="#"><span class="textlabel">GalaxytoolNG</span></a>';
+    menuGTNG.addEventListener('click', function(e) {
+        toogleMenu();
+    },false);
     menuTableTools.appendChild(menuGTNG);
+    inhalt = doc.getElementById("buttonz");
```

Comment:
__[eslint]__
* Error - "inhalt" is not defined. (no-undef)
* Error - "doc" is not defined. (no-undef)